cmd/snap/cmd_run.go

```diff
@@ -1077,6 +1078,67 @@ func (x *cmdRun) runSnapConfine(info *snap.Info, securityTag, snapApp, hook stri
 		return env.ForExecEscapeUnsafe(snapenv.PreservedUnsafePrefix)
 	}
 
+	// Systemd automatically places services under a unique cgroup encoding the
+	// security tag, but for apps and hooks we need to create a transient scope
+	// with similar purpose ourselves.
+	//
+	// The way this happens is as follows:
+	//
+	// 1) Services are implemented using systemd service units. Starting an
+	// unit automatically places it in a cgroup named after the service unit
```

Comment:
Hmm, pretty sure it's `a unit`, `an unit` does not sound right (cc @degville)
```suggestion
	// 1) Services are implemented using systemd service units. Starting a
	// unit automatically places it in a cgroup named after the service unit
```

Reply:
@bboozzoo you're right. It should be `Starting a unit...`. English is so stupid sometimes.